src/qute/commands/registerCommands.ts

```diff
@@ -13,6 +14,19 @@ export function registerVSCodeQuteCommands(context: ExtensionContext) {
   registerGenerateTemplateFileCommand(context);
   registerJavaDefinitionCommand(context);
   registerConfigurationUpdateCommand(context);
+
+  context.subscriptions.push(commands.registerCommand(QuteSettings.QUTE_VALIDATION_ENABLED + QuteSettings.TOGGLE_ON, ()=> {
+    // workspace.getConfiguration().update(QuteSettings.QUTE_VALIDATION_ENABLED, false);
+    workspace.getConfiguration().update(QuteSettings.QUTE_VALIDATION_ENABLED, !workspace.getConfiguration().get(QuteSettings.QUTE_VALIDATION_ENABLED));
```

Comment:
I'm aware of https://github.com/redhat-developer/vscode-quarkus/issues/448#issuecomment-1018703205, for now it's a PoC that the option can be toggled.